packages/mdc-toolbar/README.md

```diff
@@ -72,47 +131,81 @@ toolbar will not overlay any of the element's content.
 </main>
 ```
 
-### Sections
+### Fixed Last Row Toolbar (Requires JavaScript)
 
-Toolbar sections are aligned to the toolbar's center. You can change this
-behavior by applying `mdc-toolbar__section--align-start` or
-`mdc-toolbar__section--align-end` to align the sections to the start or the end
-of the toolbar (respectively).
+Toolbars support anchoring only the last row to the top.
 
 ```html
-<header class="mdc-toolbar">
+<header class="mdc-toolbar mdc-toolbar--fixed-lastrow-only">
   <div class="mdc-toolbar__row">
-    <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-      Section aligns to start.
-    </section>
-    <section class="mdc-toolbar__section">
-      Section aligns to center.
-    </section>
-    <section class="mdc-toolbar__section mdc-toolbar__section--align-end">
-      Section aligns to end.
-    </section>
+    <!-- This row will scroll off screen -->
+  </div>
+  <div class="mdc-toolbar__row">
+    <!-- This row will anchor on top of screen -->
   </div>
 </header>
 ```
 
-Toolbar sections are laid out using flexbox. Each section will take up an equal
-amount of space within the toolbar.
+### Flexible Toolbar (Requires JavaScript)
 
-### Toolbar title
+Flexible behavior can be added to mdc-toolbar, whose height changes as the user
+scrolls. Flexible behavior is highly customizable - we only define the change of
+flexible space size without making further assumptions. But we do recommend the
+height of flexible space should be an integral number of `mdc-toolbar__row`
+height and provide a easier way for user to customize height. User can adjust the
+height of flexible space through sass variable `$mdc-toolbar-ratio-to-extend-flexible`
+or css variable `--mdc-toolbar-ratio-to-extend-flexible`.
 
-You can use the `mdc-toolbar__title` element to style toolbar text representing
-a page's title, or an application name.
+Flexible toolbar emits `change` event and percentage of remaining flexible space
```

Comment:
How about: 

"Flexible toolbars emit a `change` custom event with a `detail` object containing `flexibleExpansionRatio` property. The `flexibleExpansionRatio` property is a number from 0-1 representing the _ratio of flexible space that has already been collapsed divided by the total amount of flexible space_".

I also think showing a script snippet showing how to use this `flexibleExpansionRation` would be really useful